dd-sdk-android/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -0,0 +1,95 @@
+//
+// Datadog Native Library.
+//
+#include <jni.h>
+#include <sstream>
+#include <cstring>
+#include <fstream>
+#include "utils/signal-monitor.h"
+#include "utils/backtrace-handler.h"
+#include "datadog-native-lib.h"
+#include "utils/datetime.h"
+#include "utils/fileutils.h"
+
+static const char *STORAGE_DIR;
```

Comment:
They're actually being set from in the `registerSignalHandler` JNI method which then registers the signal handler. Do you think it would be better to put all of these in a `struct` and make that `struct` instance static in order to be able to check if is not null easier later ?